Include missing in blob.hpp to compile solver.cpp

After last code update, compilation fails on Debian testing for the following reason in file include/caffe/blob.hpp :
```
CXX src/caffe/solver.cpp
In file included from ./include/caffe/net.hpp:10:0,
                 from src/caffe/solver.cpp:7:
./include/caffe/blob.hpp:13:26: error: ‘INT_MAX’ was not declared in this scope
 const int kMaxBlobAxes = INT_MAX;
```
This problem can be solved easily by adding the missing include
```
 #include <climits>
```
However, it should be more elegant to add this include into another header already included in this file
(maybe in caffe/common.hpp ?).

Thank you for this great library !

--- a/include/caffe/common.hpp
+++ b/include/caffe/common.hpp
@@ -5,6 +5,7 @@
 #include <gflags/gflags.h>
 #include <glog/logging.h>
 
+#include <climits>
 #include <cmath>
 #include <fstream>  // NOLINT(readability/streams)
 #include <iostream>  // NOLINT(readability/streams)

